crawlers/currency_price_tw.py

```diff
@@ -50,6 +60,8 @@ def crawl_data(self, instrument):
                 break
             for row in rows:
                 grids = list(row)
+                if not (is_float(grids[3].text) and is_float(grids[4].text)):
+                    continue
```

Comment:
If date is still available, maybe we can still make it a row as (date, None, None).